tests/entrypoints/openai/reasoning_parsers/test_deepseekr1_reasoning_parser.py

```diff
@@ -72,6 +72,35 @@
     "reasoning_content": None,
     "content": "This is the rest",
 }
+THINK_NO_END = {
+    "output": "<think>This is a reasoning section",
+    "reasoning_content": "This is a reasoning section",
+    "content": None,
+}
+EMPTY = {
+    "output": "",
+    "reasoning_content": "",
+    "content": None,
+}
+EMPTY_STREAMING = {
+    "output": "",
+    "reasoning_content": None,
+    "content": None,
+}
+NEW_LINE = {
+    "output": "\n<think>This is a reasoning section</think>\nThis is the rest",
```

Comment:
I think it is why the previous regex based approach fails sometimes. The old chat template may return `\n<think>...`.